tools/cldr-code/src/main/java/org/unicode/cldr/test/CheckCLDR.java

```diff
@@ -1117,6 +1119,44 @@ public static boolean hasType(List<CheckStatus> result, Type type) {
             }
             return false;
         }
+
+        /**
+         * @returns true if this status applies to the entire locale, not a single path
+         */
+        public boolean isNonPathBased() {
```

Comment:
Best to avoid negative names. Maybe isEntireLocale()?

Reply:
The report name might also want to be something similar… Overall problems, since **supplemental** will not mean anything to normal people :)

Reply:
Maybe "Overall Locale Errors" for the report?